testing/endtoend/components/eth1/helpers.go

```diff
@@ -46,7 +46,11 @@ func WaitForBlocks(web3 *ethclient.Client, key *keystore.Key, blocksToWait uint6
 	finishBlock := block.NumberU64() + blocksToWait
 
 	for block.NumberU64() <= finishBlock {
-		spamTX := types.NewTransaction(nonce, key.Address, big.NewInt(0), params.SpamTxGasLimit, big.NewInt(1e6), []byte{})
+		gasPrice, err := web3.SuggestGasPrice(context.Background())
```

Comment:
This was a bug fix which was causing transactions to be backlogged in the mempool. The gas price was set too low which caused issues with packing in transactions, this has now been fixed. In a follow up PR we can re-enable our transaction gen evaluator again